go.mod

```diff
@@ -30,7 +30,7 @@ require (
 	github.com/opentracing/opentracing-go v1.1.0
 	github.com/phayes/freeport v0.0.0-20180830031419-95f893ade6f2
 	github.com/pingcap/badger v1.5.1-0.20200810065601-8c92a97807f9
-	github.com/pingcap/br v0.0.0-20200820083933-d9d6207c0aa7
+	github.com/pingcap/br v0.0.0-20200915060630-cc1a3924f33e
```

Comment:
why update this?

Reply:
Refer to https://github.com/pingcap/br/pull/503